lib/app/features/competiton/competition_widgets/competition_search_bar.dart

```diff
@@ -6,56 +6,74 @@ import 'package:score_live/app/features/competiton/cubit/competition_cubit.dart'
 import 'package:score_live/core/applocalization_context.dart';
 import 'package:score_live/presentation/constants/app_colors.dart';
 import 'package:score_live/presentation/constants/app_const_variables.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
 
 class CompetitionSearchBar extends StatelessWidget {
   CompetitionSearchBar({super.key});
 
+  final List<String> searchTypes = <String>['Team', 'League (name)'];
```

Comment:
What is the purpose of this?

Reply:
this list is provided as a dropdown button options later in the code, I didn't want them to be coded inside the widget so I extracted whole list outside of build method

Reply:
So If the list does not change than I would keep such a list in separate class with constant values or something else but not in the widget itself

Reply:
good point